contracts/token/Minter.sol

```diff
@@ -171,7 +171,7 @@ contract Minter is Manager, IMinter {
     /**
      * @notice Deposit ETH to this contract. Only callable by the currently registered Minter or JobsManager
      */
-    function depositETH() external payable onlyMinterOrJobsManager whenSystemNotPaused returns (bool) {
+    function depositETH() external payable onlyMinterOrJobsManager returns (bool) {
```

Comment:
This change is necessary otherwise since `migrateToNewMinter()` can only be called when the Controller is paused. Without this change, the call to `depositETH()` in `migrateToNewMinter()` would revert.

Worth noting that this requirement was not obvious (at least I missed it) until the integration test in 78d22e2 was added. The Minter unit test cases for `migrateToNewMinter()` deploy a [GenericMock](https://github.com/livepeer/protocol/blob/826af1124f645a819b54eb0500ddc31f0d2af755/test/unit/Minter.js#L141) as the new Minter which mocks the `depositETH()` implementation such that it would never revert. However, the `depositETH()` implementation prior to this change would revert if the Controller was paused.

Removing the `whenSystemNotPaused` modifier here should be safe because the only two contracts that can call `depositETH()` are the Minter and the TicketBroker (registered under the JobsManger contract ID). If the caller is the Minter, then we would be running a migration meaning that the Controller needs to be paused. The TicketBroker already has a `whenSystemNotPaused` modifier applied to the functions that invoke `depositETH()`.

Reply:
For the record, this problem would have already existed prior to this upgrade right ? 

Would an alternative be removing the paused requirement for calling `migrateToNewMinter()` ? 

The pausing requirement seems like a weird thing in a case where we would have LIP-25, but I can see how it could still be preferred.

Reply:
> For the record, this problem would have already existed prior to this upgrade right ?

Yep.

> Would an alternative be removing the paused requirement for calling migrateToNewMinter() ?

That doesn't help here because we would be calling `migrateToNewMinter()` on the existing Minter contract which already has a paused requirement.

Reply:
LIP-40 has been updated in livepeer/LIPs#47 to reflect these changes.

Reply:
The boolean return value seems unnecessary but since this existed in the previous Minter contract as well I don't think we should change it. Just an observation, but it would reduce the bytecode size by a tiny amount.